src/Components/MainHeader/MiddleMenu/Function/Calender/Calender.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Calender.css';
+
+function CalenderModal() {
+  return (
+    <div className="calender-modal-outter-wrapper">
+      <div className="calender-modal-inner-wrapper">
+        <div className="calender-switch-area-wrapper">
+          <div className="calender-switch-inner-btn-area">
+            <button className="calender-switch-btn-common-attr-A">
```

Comment:
```suggestion
            <button className="calender-switch-btn-common-attr-어떤기능">
```
이런식으로 직접 기능을 명시하는 것이 A, B라고 하는 것보다 좋을 것 같아요